src/test/java/org/openapitools/client/api/MessagesApiTest.java

```diff
@@ -69,67 +66,71 @@ public class MessagesApiTest {
+                messageDirection, carrierName, messageType, errorCode, fromDateTime, toDateTime, sort, pageToken,
+                limit);
+
+        assertThat(response, instanceOf(MessagesList.class));
+        assertThat(response.getTotalCount(), greaterThan(0));
+        assertThat(response.getMessages().get(0), instanceOf(ListMessageItem.class));
+        assertThat(response.getMessages().get(0).getAccountId(), is(System.getenv("BW_ACCOUNT_ID")));
+        assertThat(response.getMessages().get(0).getDestinationTn(), matchesRegex("^\\+[1-9]\\d{1,14}$"));
+        assertThat(response.getMessages().get(0).getMessageDirection(), instanceOf(ListMessageDirectionEnum.class));
+        assertThat(response.getMessages().get(0).getMessageId(), matchesRegex("^.+$"));
+        assertThat(response.getMessages().get(0).getMessageStatus(), instanceOf(MessageStatusEnum.class));
+        assertThat(response.getMessages().get(0).getMessageType(), instanceOf(MessageTypeEnum.class));
+        assertThat(response.getMessages().get(0).getSegmentCount(), greaterThan(0));
     }
 
-
     @Test
     public void listMessageBadRequestTest() {
 
-      Basic.setUsername(BW_USERNAME);
-      Basic.setPassword(BW_PASSWORD);
-      String pageToken = "gdEewhcJLQRB5"; // Bad Token
+        Basic.setUsername(BW_USERNAME);
+        Basic.setPassword(BW_PASSWORD);
+        String pageToken = "gdEewhcJLQRB5"; // Bad Token
 
-      ApiException exception = Assertions.assertThrows(ApiException.class, () -> api.listMessages(accountId, messageId, sourceTn, destinationTn, messageStatus, messageDirection, carrierName, messageType, errorCode, fromDateTime, toDateTime, sort, pageToken, limit));
-      assertThat(exception.getCode(), is(400));
+        ApiException exception = Assertions.assertThrows(ApiException.class,
+                () -> api.listMessages(accountId, messageId, sourceTn, destinationTn, messageStatus, messageDirection,
+                        carrierName, messageType, errorCode, fromDateTime, toDateTime, sort, pageToken, limit));
+        assertThat(exception.getCode(), is(400));
 
     }
-    
+
     @Test
     public void listMessageUnauthorizedTest() {
 
-      Basic.setUsername("user");
-      Basic.setPassword("pass");
-
-      ApiException exception = Assertions.assertThrows(ApiException.class, () -> api.listMessages(accountId, messageId, sourceTn, destinationTn, messageStatus, messageDirection, carrierName, messageType, errorCode, fromDateTime, toDateTime, sort, pageToken, limit));
-      assertThat(exception.getCode(), is(401));
-    }        
-      /**
-       * Create Message
-       *
-       * Endpoint for sending text messages and picture messages using V2 messaging.
-       *
-       * @throws ApiException if the Api call fails
-       */
-
-      @Test
-      public void createMessageTest() throws ApiException {
+        Basic.setUsername("user");
+        Basic.setPassword("pass");
```

Comment:
```suggestion
        Basic.setUsername("bad_username");
        Basic.setPassword("bad_password");
```
just because these are what you used in the TN lookup tests, I think these should be somewhat consistent even if it doesn't really matter